src/app/main.cpp

```diff
@@ -39,6 +41,34 @@ void HandleStop(int i) {
   }
 }
 
+// Print a path warning if the running directory is NOT one level above the binary directory
+void printPathWarning() {
+  char cwd[PATH_MAX];
+  getcwd(cwd, PATH_MAX);
```

Comment:
The return value of getcwd needs to be checked for failure:
https://linux.die.net/man/3/getcwd